docs/scanner-pools.md

```diff
@@ -0,0 +1,182 @@
+
+## Disabling existing nodes
+
+If a node is no longer used, it should be disabled to free up allocated stake for newer nodes.
+
+- Navigate to the node pool that has the node you want to disable
+
+- Select the disable action on the node you want to disable
+
+![Disable Scan Node](DisableScanNode.png)
+
+## Adjusting stake allocation
+
+While it's always possible to add more stake to a pool, it is allocated proportionally to the amount of scan nodes in the pool. Total allocated stake can only be `node_count × max_stake_per_scan_node`.
+
+For example, if there are two scan nodes in a pool, a total of 35k FORT deposited and only 15k FORT worth of max stake allocation is allowed per scan node, then the pool can allocate a maximum of `15k × 2 = 30k` FORT in total. This is the amount that is effective in calculating the rewards and the rest of `35k - 30k = 5k` remains unallocated until more scan nodes are registered to the pool.
+
+It is up to the pool owner to allocate more from owner stake or more from the delegators so the share of the rewards can be adjusted accordingly.
+
+For each node in a pool:
+
+- The operator needs to allocate at least 2500 FORT.
+- The total capacity is 15000 FORT.
+
+When you deposit more stake on your pool as the operator and then register a node, your stake is allocated automatically. It is up to you to decide how much of owner and delegated stake to be allocated.
+
+!!! warning "Risk of shutting down the pool"
+    After unallocating owner stake as the operator, if allocated owner stake does not satisfy `node_count × min_stake_per_scan_node`, the pool faces the risk of shutting down and will generate no rewards! Please take into account the warnings and your calculations before proceeding.
+
+    E.g. for a pool with two nodes, 4k FORT stake on the pool means that both of the nodes in the pool are non-operational until it can be topped up to 5k (`2500 × 2`) and allocation of owner stake is adjusted accordingly.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to adjust the stake allocation
+
+![My Node Pools List](MyNodePoolsList.png)
+
+- Click on "Allocate Stake"
+
+![Allocate Stake On Node](AllocateStakeOnNode.png)
+
+- From here you can either allocate or unallocate stake. To adjust owner stake make sure the subject is *Owner*
+
+![Allocate Modal](AllocationModal.png)
+
+- To adjust the stake of delegators select the *Delegators* subject
+
+![AllocateDelegates](AllocateDelegates.png)
+
+## Changing the commission
+
+In addition to the allocation adjustment, operators can decide how much of the delegators' rewards they would like to keep as commission.
+
+!!! important "Phase II feature"
+    These commissions will be effective when Forta App supports claiming the rewards. Meanwhile, the pool owners can still expect delegators to stake on their pools and adjust allocation and commission to get ready for the second phase.
```

Comment:
the commissions will be effective regardless of what is in Forta App. i think we should remove this block